### DISCUSSION TITLE
Asset names have an unnecessary period at v2.30.0

### DISCUSSION THREAD

**Discussion by suzuki-shunsuke** [goreleaser/nfpm#673] [Q&A] [answered]
Thank you for your great project.
This is a bug report of asset names in GitHub Releases.

The following assets have an unnecessary period between `nfpm_` and `2.30.0`.

https://github.com/goreleaser/nfpm/releases/tag/v2.30.0

```
nfpm_.2.30.0_Darwin_arm64.tar.gz
nfpm_.2.30.0_Darwin_arm64.tar.gz.sbom
nfpm_.2.30.0_Darwin_x86_64.tar.gz
nfpm_.2.30.0_Darwin_x86_64.tar.gz.sbom
nfpm_.2.30.0_Linux_arm64.tar.gz
nfpm_.2.30.0_Linux_arm64.tar.gz.sbom
nfpm_.2.30.0_Linux_x86_64.tar.gz
nfpm_.2.30.0_Linux_x86_64.tar.gz.sbom
nfpm_.2.30.0_Windows_arm64.zip
nfpm_.2.30.0_Windows_arm64.zip.sbom
nfpm_.2.30.0_Windows_x86_64.zip
nfpm_.2.30.0_Windows_x86_64.zip.sbom
```

At v2.30.0, `.goreleaser.yml` was changed, but the change looks no problem. 🤔

https://github.com/goreleaser/nfpm/compare/v2.29.0...v2.30.0

**Comment by caarlos0** [comment]
Hmmm will investigate, thank you!

**Reply by suzuki-shunsuke** [reply]
I sent a pull request. https://github.com/goreleaser/nfpm/pull/674

> At v2.30.0, .goreleaser.yml was changed, but the change looks no problem.

I overlooked the bug of the change.

**Reply by suzuki-shunsuke** [reply, marked as answer]
https://github.com/goreleaser/nfpm/pull/674 was merged and v2.30.1 was out. The bug has been solved.
https://github.com/goreleaser/nfpm/releases/tag/v2.30.1

**Reply by caarlos0** [reply]
thanks for reporting and gixing it, @suzuki-shunsuke 💜
